Tiltfile

```diff
@@ -252,18 +253,37 @@ def build_docker_image(image, context, binary_name, additional_docker_build_comm
 
     # Set up an image build for the provider. The live update configuration syncs the output from the local_resource
     # build into the container.
-    docker_build(
-        ref = image,
-        context = context + "/.tiltbuild/bin/",
-        dockerfile_contents = dockerfile_contents,
-        build_args = {"binary_name": binary_name},
-        target = "tilt",
-        only = binary_name,
-        live_update = [
-            sync(context + "/.tiltbuild/bin/" + binary_name, "/" + binary_name),
-            run("sh /restart.sh"),
-        ],
-    )
+    if settings.get("build_engine") == "podman":
+        bin_context = context + "/.tiltbuild/bin/"
```

Comment:
nit 
```suggestion
        bin_context = context + "/.tiltbuild/docker/"
```

Reply:
Yes it was simplier to have the `Dockerfile` at the same place to have the `manager` binary in the context.

By moving the `Dockerfile` elsewhere and setting the context, it doesn't  find the `manager` binary for some reason.
I can investigate more if needed.

Reply:
I fixed the issue I had with the context, I can propose this alternative to have the Dockerfile in a seperate path:

```diff
@@ -255,15 +260,16 @@ def build_docker_image(image, context, binary_name, additional_docker_build_comm
     # build into the container.
     if settings.get("build_engine") == "podman":
         bin_context = context + "/.tiltbuild/bin/"
-        local("tee %s/Dockerfile" % (shlex.quote(bin_context)), quiet = True, stdin = dockerfile_contents)
+        dockerfile_path = context + "/.tiltbuild/docker/"
+        local("mkdir -p %s;tee %s/Dockerfile" % (shlex.quote(dockerfile_path), shlex.quote(dockerfile_path)), quiet = True, stdin = dockerfile_contents)

         custom_build(
             ref = image,
             command = (
                 "set -ex\n" +
-                "podman build -t $EXPECTED_REF --build-arg binary_name=%s --target tilt %s \n" +
+                "podman build -t $EXPECTED_REF --build-arg binary_name=%s --target tilt -f %s/Dockerfile %s \n" +
                 "podman push --format=docker $EXPECTED_REF\n"
-            ) % (binary_name, shlex.quote(bin_context)),
+            ) % (binary_name, shlex.quote(dockerfile_path), shlex.quote(bin_context)),
             deps = [bin_context],
             skips_local_docker = True,
             live_update = [
```

Reply:
kk, let's keep it simple as it is, but please add a comment documenting that we put the docker file where the manager binary is so it is in the default context.

Reply:
As suggested, I just added a comment